gobblin-runtime/src/test/java/gobblin/runtime/JobStateStoreTest.java

```diff
@@ -111,7 +109,7 @@ public void testLaunchSecondJob() throws Exception {
   public void testLaunchThirdJob() throws Exception {
     Closer closer = Closer.create();
     try {
-      closer.register(new LocalJobLauncher(this.properties, this.jobProps)).launchJob(null);
+      closer.register(new LocalJobLauncher(this.jobConfig)).launchJob(null);
```

Comment:
Due to the refactoring above, the single `jobConfig` already includes everything so it makes more sense to use `LocalJobLauncher` directly.